src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  /* max-height: 60px; */
+  margin: auto;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+  padding: 30px 0 ;
```

Comment:
Wrong paddings, so this is the reason why your tests do not pass. Take padding from mockup and and round them up